corona-warn-app/cwa-app-android
# Restrict number persons warning dialog (EXPOSUREAPP-10970)

## Whats new
Warning dialogs if number of persons is more than threshold and maximum count of persons certificates.

## Jira
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10970)

## How to test
Create and scan certificates for new person every time of any type until you would see warning dialog.
